frontend/src/booking/booking-modal.vue

```diff
@@ -10,40 +10,47 @@
            hide-header
            size="md">
     <div v-if="showModal">
-      <b-container style="font-size:1.1rem; border:1px solid lightgrey; border-radius: 10px">
+      <b-container style="font-size:1.1rem; border:1px solid lightgrey; border-radius: 10px" class="mb-2 pb-3">
         <b-row>
-          <b-col><b>Booking Details</b></b-col>
+          <b-col><b>Step 1: Pick an Invigilator</b></b-col>
         </b-row>
         <b-row no-gutters>
-          <b-col cols="4">Date: </b-col><b-col> {{ date.start.local().format('ddd, MMM Do, YYYY')}}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">Room: </b-col><b-col> {{ date.resource.title }}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">Start: </b-col><b-col> {{ date.start.local().format('h:mm a') }}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">End: </b-col><b-col> {{ endTime.local().format('h:mm a') }}</b-col>
-        </b-row>
-      </b-container>
-      <b-container class="p-3" style="font-size: .9rem;">
-        <b-row no-gutters>
-          <b-col cols="4" >Exam: </b-col><b-col> {{ exam.exam_name }}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">Event ID: </b-col><b-col> {{ exam.event_id }}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">Type: </b-col><b-col> {{ exam.exam_type.exam_type_name }}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">Format: </b-col><b-col> {{ exam.exam_method }}</b-col>
-        </b-row>
-        <b-row no-gutters>
-          <b-col cols="4">Duration: </b-col><b-col> {{ exam.exam_type.number_of_hours+' hrs' }}</b-col>
+          <b-col cols="4">Invigilator: </b-col><b-col> <b-form-select v-model="selected"
```

Comment:
This variable should be a little more aptly named... `selectedInvigilator`, `scheduledInvigilator` or something?

Reply:
changed to selected invigilator on lines 18, 23, 111, 121